app/gcp/iam.tf

```diff
@@ -9,7 +9,6 @@ resource "google_service_account" "cloud_run_svc" {
 resource "google_project_iam_member" "cloud_run_svc_roles" {
   for_each = toset([
     "roles/secretmanager.secretAccessor",
-    "roles/run.admin",           # To manage Cloud Run services
```

Comment:
we do need this role, I'm open to narrow the role, but then we need to create a replacement and tests should pass